tests/acceptance/test_literature_suggestion_form.py

```diff
@@ -132,16 +132,17 @@ def test_literature_create_article_journal_with_proceeding_manually(login):
 
 def _check_back_office(input_data):
     holding_panel_literature_list.go_to()
-    assert holding_panel_literature_list.load_submission_record(
-        input_data
-    ).has_error()
+    assert holding_panel_literature_list.load_submitted_record().has_error()
 
     holding_panel_literature_detail.go_to()
     assert holding_panel_literature_detail.load_submitted_record(
         input_data
     ).has_error()
     assert holding_panel_literature_detail.accept_record().has_error()
 
+    holding_panel_literature_list.go_to()
+    assert holding_panel_literature_list.load_completed_record().has_error()
+
```

Comment:
This lines are checking if the user is able to see the completed record in the UI